examples/react/TextHighlighter/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "carbon-labs-react-text-highlighter",
```

Comment:
We could use the `@carbon-labs` prefix here, so a name could be:
`@carbon-labs/react-text-highlighter`